cluster-up/cluster/kind-1.22-sriov/sriov-components/manifests/sriov-cni-daemonset.yaml

```diff
@@ -18,19 +18,23 @@ spec:
         tier: node
         app: sriov-cni
     spec:
-      hostNetwork: true
       nodeSelector:
-        beta.kubernetes.io/arch: amd64
+        kubernetes.io/arch: amd64
       tolerations:
-      - key: node-role.kubernetes.io/control-plane
+      - key: node-role.kubernetes.io/master
```

Comment:
a toleration for control-plane might still be required here? and in the sriovdp-daemonset.yaml